### PR TITLE
【課題解決Wiki】【実務 / Unity】プレイヤーに追従するテキストの実装

### PR DESCRIPTION
**Pull request by akiRAM2**
### **PR Type**
Documentation

___

### **Description**
- 新しいガイド「プレイヤーに追従するテキストの実装」を英語と日本語で追加しました。
- ガイドには、実装手順、環境情報、コード例、およびカスタマイズに関する追加情報が含まれています。
- 英語および日本語の変更ログに新しいガイドの情報を追加しました。
- mkdocs.ymlに新しいガイドのナビゲーションエントリを追加しました。

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>PlayerFollowText.en.md</strong><dd><code>Added English guide for player-following text implementation.</code></dd></summary>
<hr>

docs/WorldMakingGuide/PlayerFollowText.en.md

<li>Added a new guide in English for implementing text that follows the <br>player.<br> <li> Included implementation steps, environment details, and code examples.<br> <li> Provided additional notes for customization.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/570/files#diff-cf524fbf7a37d07f7d4dd34c3fbf9941e5ee0cc456a3d8abee1aaf3b21833a6d">+55/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>PlayerFollowText.ja.md</strong><dd><code>Added Japanese guide for player-following text implementation.</code></dd></summary>
<hr>

docs/WorldMakingGuide/PlayerFollowText.ja.md

<li>Added a new guide in Japanese for implementing text that follows the <br>player.<br> <li> Included implementation steps, environment details, and code examples.<br> <li> Provided additional notes for customization.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/570/files#diff-8cd83748c6d6b9f98189744a65d17a8f1c116a38e48654e1d678696f675a6883">+56/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>changelog-14.2.en.md</strong><dd><code>Updated English changelog for new guide.</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docs/changelog/changelog-14.2.en.md

<li>Updated the changelog to include the new English guide for <br>player-following text.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/570/files#diff-2f3baf40cb41ed78decf8d39000daef3a13e7b63f50de72c9d8a685e1356d6bb">+9/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>changelog-14.2.ja.md</strong><dd><code>Updated Japanese changelog for new guide.</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docs/changelog/changelog-14.2.ja.md

<li>Updated the changelog to include the new Japanese guide for <br>player-following text.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/570/files#diff-a8f7442620adccd75c20db969e01eb0df04d1ae71c0bdda20f51ce3726dd1a5b">+9/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>mkdocs.yml</strong><dd><code>Updated navigation for new player-following text guide.</code>&nbsp; &nbsp; </dd></summary>
<hr>

mkdocs.yml

<li>Added navigation entries for the new player-following text guide in <br>both English and Japanese.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/570/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: docs/WorldMakingGuide/PlayerFollowText.ja.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# プレイヤーに追従するテキストの実装
+
+## 概要
+
+プレイヤーに追従するテキストの実装方法です。
+
+![PlayerFollowText](img/PlayerFollowText_01.jpg)
+
+!!! info "検証環境"
+    SDKバージョン : SDK5.4.0
```
Comment (ra9g16) [inline docs/WorldMakingGuide/PlayerFollowText.ja.md:10]: <br>を入れてあげないと改行されないかもです!
image

**Review by ra9g16 — CHANGES_REQUESTED** [review]
修正点ひとつあります！

**Review by Kondo-Totori — APPROVED** [review]
LGTM!

**Review by ra9g16 — APPROVED** [review]
LGTM!

**Review by Negi-T — APPROVED** [review]
LGTM!

**Pull request merged**